web/src/NetworksList.jsx

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright (c) [2022] SUSE LLC
+ *
+ * All Rights Reserved.
+ *
+ * This program is free software; you can redistribute it and/or modify it
+ * under the terms of version 2 of the GNU General Public License as published
+ * by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful, but WITHOUT
+ * ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+ * FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+ * more details.
+ *
+ * You should have received a copy of the GNU General Public License along
+ * with this program; if not, contact SUSE LLC.
+ *
+ * To contact SUSE LLC about this file by physical or electronic mail, you may
+ * find current contact information at www.suse.com.
+ */
+
+import React from "react";
+
+import { ConnectionState } from "./client/network/model";
+import NetworkListItem from "./NetworkListItem";
+
+function NetworksList({ networks, activeNetwork, selectedNetwork, onSelectionCallback, onCancelSelectionCallback }) {
+  const isStateChanging = (network) => {
+    const state = network.connection?.state;
+    return state === ConnectionState.ACTIVATING || state === ConnectionState.DEACTIVATING;
+  };
+
+  return networks.map(n => {
+    const isChecked = n.ssid === selectedNetwork?.ssid;
+    const showAsChecked = !selectedNetwork && n.ssid === activeNetwork?.ssid;
```

Comment:
np: Would not be better something like `isActive`? I meant, the component will decide whether to show it. It sounds more "semantic" (perhaps `isSelected` and `isActive`). Just a suggestion, feel free to ignore it.

Reply:
Completely agree. Not an _np_ at at all.

Believe it or not, I thought the same after reading the code, a couple of hours after writing it.

Definitely, being in a rush is not a good thing ;)

Thanks a lot for taking your time for carefully reviewing that PR.